mreg/apps.py

```diff
@@ -1,8 +1,22 @@
 from django.apps import AppConfig
+from django.conf import settings
+from django.db.models.signals import post_migrate
 
 
 class MregAppConfig(AppConfig):
     name = 'mreg'
 
     def ready(self):
         import mreg.signals # noqa
+        from .models.network_policy import NetworkPolicyAttribute
```

Comment:
Could be an absolute import like most other imports.

```suggestion
        from mreg.models.network_policy import NetworkPolicyAttribute
```

Reply:
Yep. Done.